src/cli/exec.rs

```diff
@@ -59,6 +59,10 @@ pub struct Args {
     #[clap(long = "list", num_args = 0..=1, default_missing_value = "", require_equals = true)]
     pub list: Option<String>,
 
+    /// Disable modification of the PS1 prompt to indicate the temporary environment
+    #[clap(long, action = clap::ArgAction::SetTrue, default_value = "false")]
```

Comment:
```suggestion
    #[clap(long)]
```

That should already be the default